brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaPlanToSpecTransformerIntegrationTest.java

```diff
@@ -397,6 +397,26 @@ public boolean apply(@Nullable Entity entity) {
         });
     }
 
+    @Test
+    public void testResolvesKeywordInFunction() {
```

Comment:
- I think we should aim for more tests that take milliseconds even if it means mocking Alien components.